REFPLTB-1474: Task to stabilize opensync 2.4.5 on Turris omnia and upstream it to meta-turris

rnaren94 opened the pull request:
REFPLTB-1474: stabilize opensync 2.4.5 on Turris omnia

rnaren94 committed:
EXTLTTS_59:Task to stabilize opensync 2.4.5 on Turris omnia and upstream it to meta-turris

Signed-off-by: rnaren94 <r.naren@ltts.com>

rnaren94 committed:
delete opensync 2 0 5 bbappend file

rdkcmf-jenkins commented:
https://jenkins.cmf.code.rdkcentral.com/job/github-triggered/7282/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered/12420/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/42806/ started.

rdkcmf-jenkins commented:
build-tur-ext Console Log uploaded to Gist URL: https://gist.github.com/4d46dae9ee0af29504d7a6eab3230bd2

rdkcmf-jenkins commented:
build-tur-bb Console Log uploaded to Gist URL: https://gist.github.com/c8f96bd3c23e5550a86519273838e021

manigandanj commented on recipes-connectivity/opensync/opensync_2.4.5.bbappend:
```diff
@@ -0,0 +1,44 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/opensync_2.4.5:${THISDIR}/files:"
+#FILESEXTRAPATHS_prepend := "${THISDIR}/opensync_2.4.5:${THISDIR}/files:"
```
please remove this line.

manigandanj replied:
please remove this line.

manigandanj reviewed (changes requested):
Don't remove this opensync_2.0.5.bbappend until meta-rdk-opensync in manifest XML is updated to point to osync_2.4.5

rnaren94 committed:
edited out commented lines in bbappend

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered/12460/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/43027/ started.

rdkcmf-jenkins commented:
https://jenkins.cmf.code.rdkcentral.com/job/github-triggered/7316/ started

rdkcmf-jenkins commented:
build-tur-ext Console Log uploaded to Gist URL: https://gist.github.com/c1edd55fa6f32ac6a8f94eb473f90f39

rnaren94 committed:
added 2.0.5 bbappend file 

till 2.4.5 is updated in xml

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered/12462/ started

rdkcmf-jenkins commented:
https://jenkins.cmf.code.rdkcentral.com/job/github-triggered/7317/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/43028/ started.

rdkcmf-jenkins commented:
## Blackduck scan failure details

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/meta-turris/371/rdkcentral/meta-turris

* Commit: 876fa14e1380a49e82b26be1a3a367e5c3505ea2

 **Report detail:** [gist](https://gist.github.com/b219e4bfee4b20d2074cc38e931237bd)

rnaren94 committed:
delete opensync 2.0.5

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered/12464/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/43029/ started.

rdkcmf-jenkins commented:
https://jenkins.cmf.code.rdkcentral.com/job/github-triggered/7319/ started

rnaren94 committed:
added 2.0.5 bbappend

needs to be available till 2.4.5 is updated in manifest xml

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered/12465/ started

rdkcmf-jenkins commented:
https://jenkins.cmf.code.rdkcentral.com/job/github-triggered/7320/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/43030/ started.

rdkcmf-jenkins commented:
Smoke tests for ['build-tur-bb'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/103797/

scthunderbolt commented:
Please have a meaningful subject line in the commit-msg.

rnaren94 committed:
REFPLTB-1474: Stabilize opensync 2.4.5 on Turris omnia

Reason for change: added bbappend file and associated patch files to stabilize build for opensync 2.4.5
Test procedure: Built image and verified device is online in FLT
Risks: Low

rdkcmf-jenkins commented:
https://jenkins.cmf.code.rdkcentral.com/job/github-triggered/7493/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered/12628/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/43627/ started.

rdkcmf-jenkins commented:
build-tur-bb Console Log uploaded to Gist URL: https://gist.github.com/7798b6e216b423bae139b3c5c2ad8c91

rnaren94 committed:
Merge branch 'master' into opensync_2_4_5

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered/12637/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/43660/ started.

rdkcmf-jenkins commented:
https://jenkins.cmf.code.rdkcentral.com/job/github-triggered/7505/ started

rdkcmf-jenkins commented:
Smoke tests for ['build-tur-bb'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/104941/

scthunderbolt commented:
Closed due to inactivity, please restore if still needed.
